styles.sass

```diff
@@ -0,0 +1,248 @@
+---
+---
+
+
+// variables 
+
+$dark-grey: #6F6B6B;
+$light-grey: #C6CECF;
+$light-blue: #8BCBFA;
+$medium-blue: #1097FA;
+$primary-text-bold: 700;
+$primary-text-italic: italic;
+
+
+// mixins
+
+@mixin nav-menu-links
+	padding: 0;
+	text-align: center;
+	list-style-type: none;
+
+@mixin accent-text
+	font-family: acier-bat-solid, sans-serif;
+	font-weight: 400;
+	font-style: normal;
+	color: $light-grey;
+
+@mixin primary-text
+	font-family: itc-galliard, serif;
+	font-weight: 400;
+	font-style: normal;
+	font-size: 14px;
+	line-height: 1.5;
+	color: $dark-grey;
+
+@mixin link-text
+	@include accent-text
+	font-size: 14px;
+	color: $light-blue;
+	text-decoration-line: none;
+	letter-spacing: 1px;
+	margin: 2px;
+
+
+@mixin primary-header
+	@include accent-text;
+	font-size: 24px;
+
+@mixin small-screen-section-spacing
+	width: 100%
+	margin: 332px 0 332px 0;
+
+@mixin small-screen-container-spacing
+	width: 100%; 
+	margin: 0 auto;
+
+
+// element styles
+
+* 
+	box-sizing: border-box;
+
+h1, h2
+	@include primary-header;
+
+
+.container
+	width: 80%;
+	margin: 0 auto;
+	overflow: auto;
+
+.background-image-container 
+	height: 164px;
+	width: 100%;
+	opacity: 50%;
+
+	position: fixed;
+	z-index: 1;
+	bottom: 0;
+	background-image: url("images/ocean_wave.svg");
+
+
+// nav styles
+
+.main-nav
+	padding-top: 16px;
+
+.nav-menu-links
+	@include nav-menu-links;
+
+	float: left;
+
+.nav-menu-social-links
+	@include nav-menu-links;
+
+	float: right;
+
+.nav-menu-item a 
```

Comment:
We should likely create a class name for these links instead of styling links. Generally style classes > styling elements as it helps future proof us if we want to have a link that looks differently.